app/src/main/res/layout/resume_lesson_fragment.xml

```diff
@@ -64,7 +64,7 @@
         android:layout_marginTop="16dp"
         android:layout_marginEnd="32dp"
         android:ellipsize="end"
-        android:textColor="@color/component_color_resume_lesson_activity_description_color"
+        android:textColor="@color/component_color_shared_primary_text_color"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@id/resume_lesson_chapter_title_text_view" />
```

Comment:
You need to update `start_over_button_background` which is getting used in `Start Over` button.

Reply:
Done.